posts/the-radon-nikodym-theorem-in-lean.md

```diff
@@ -0,0 +1,162 @@
+---
+author: 'Kexing Ying'
+category: 'New in mathlib'
+date: 2021-09-22 12:51:14 UTC+02:00
+description: ''
+has_math: true
+link: ''
+slug: the-radon-nikodym-theorem-in-lean
+tags: ''
+title: The Radon-Nikodym theorem in Lean
+type: text
+---
+
+I have for the past two months been working on formalising the Radon-Nikodym theorem 
+in Lean, and with [PR #9065](https://github.com/leanprover-community/mathlib/pull/9065) 
+merged into mathlib, this journey seems to have finally come to an end. 
+
+The Radon-Nikodym theorem provides a necessary and sufficient condition for 
+comparing two measures, and allows us (under certain conditions) to express 
+one measure in terms of another.
+The Radon-Nikodym theorem is an important result in measure theory and has a
+wide range of applications in different fields of mathematics. Most notably, 
+it can be applied in probability theory in the definition 
+of the conditional expectation and in mathematical finance through the Girsanov 
+theorem[^1].
+
+[^1]: [wikipedia.org/wiki/Radon-Nikodym_theorem#Applications](https://en.wikipedia.org/wiki/Radon%E2%80%93Nikodym_theorem#Applications)
+
+Given two measures $\mu$ and $\nu$ on $\alpha$, we say that $\mu$ is 
+*absolutely continuous* with respect to $\nu$ (and write $\mu \ll \nu$) if for all 
+$S \subseteq \alpha$, $\nu(S) = 0$ implies $\mu(S) = 0$. Absolutely continuity 
```

Comment:
```suggestion
subset $S$ of $\alpha$, $\nu(S) = 0$ implies $\mu(S) = 0$. Absolute continuity 
```
The first edit tries to soften the discrepancy between informal math pretending to be based on set theory and Lean code.